sktime/forecasting/mapa.py

```diff
@@ -710,14 +702,7 @@ def get_test_params(cls, parameter_set="default"):
                 "decompose_type": "multiplicative",
                 "forecast_combine": "mean",
             },
-            {
```

Comment:
why are we removing this test case? It should work, from what I can see.

Reply:
If you checkout [these](https://github.com/sktime/sktime/actions/runs/13435581782/job/37537138026?pr=7872) logs, the tests fail for that param set.